lib/LaTeXML/Common/Model/RelaxNG.pm

```diff
@@ -476,7 +476,10 @@ sub simplify {
           elsif (($combination eq 'group') && ($prevc ne 'group')) {    # Use old combination!?!?!?!?
             $combination = $prevc; } }
         $$self{defs}{$qname} = simplifyCombination(['combination', $combination,
-            ($prev ? @$prev : ()), @xargs]);
+            # careful, if $prev is a triple [$op,$name,@stuff] and we flatten it,
+            # we should not treat $op and $name as @stuff.
+            # We should either drop them, or not flatten at all. Drop for now.
+            ($prev ? @$prev[2 .. $#{$prev}] : ()), @xargs]);
```

Comment:
I think the idea is that `simplifyCombination` is supposed to evolve to give a "good" description, but it clearly isn't there yet.  I expect that probably `$prev` would have been more correct than `@$prev` and better than throwing away the operator. OTOH, that leaves `simplifyCombination` to deal with the initial case where the extra items are empty (that puts a "()" into the summary).  And none of the variants are generating sensible descriptions for the tricky cases `svg, xhtml:*` and such; that ought to be our goal, I think.